slider/curve.py

```diff
@@ -131,10 +125,28 @@ def __call__(self, t):
 class Passthrough(Curve):
     kinds = 'P'
 
-    def __init__(self, points, req_length):
+    def __new__(cls, points, req_length):
```

Comment:
The changes in this module are actually unrelated and should probably have been pulled out. ping @blimmo